docs/modules/ROOT/pages/api/finance.adoc

```diff
@@ -0,0 +1,296 @@
+
+[.hljs-theme-dark]
+```cairo
+use openzeppelin_finance::vesting::interface::IVesting;
+```
+
+[.contract-index]
+.Functions
+--
+* xref:#IVesting-start[`++start()++`]
+* xref:#IVesting-cliff[`++cliff()++`]
+* xref:#IVesting-duration[`++duration()++`]
+* xref:#IVesting-end[`++end()++`]
+* xref:#IVesting-released[`++released(token)++`]
+* xref:#IVesting-releasable[`++releasable(token)++`]
+* xref:#IVesting-vested_amount[`++vested_amount(token, timestamp)++`]
+* xref:#IVesting-release[`++release(token)++`]
+--
+
+[.contract-index]
+.Events
+--
+* xref:#IVesting-AmountReleased[`++AmountReleased(token, amount)++`]
+--
+
+[#IVesting-Functions]
+==== Functions
+
+[.contract-item]
+[[IVesting-start]]
+==== `[.contract-item-name]#++start++#++() → u64++` [.item-kind]#external#
+
+Returns the timestamp marking the beginning of the vesting period.
+
+[.contract-item]
+[[IVesting-cliff]]
+==== `[.contract-item-name]#++cliff++#++() → u64++` [.item-kind]#external#
+
+Returns the timestamp marking the end of the cliff period.
+
+[.contract-item]
+[[IVesting-duration]]
+==== `[.contract-item-name]#++duration++#++() → u64++` [.item-kind]#external#
+
+Returns the total duration of the vesting period.
+
+[.contract-item]
+[[IVesting-end]]
+==== `[.contract-item-name]#++end++#++() → u64++` [.item-kind]#external#
+
+Returns the timestamp marking the end of the vesting period.
+
+[.contract-item]
+[[IVesting-released]]
+==== `[.contract-item-name]#++released++#++(token: ContractAddress) → u256++` [.item-kind]#external#
+
+Returns the already released amount for a given `token`.
+
+[.contract-item]
+[[IVesting-releasable]]
+==== `[.contract-item-name]#++releasable++#++(token: ContractAddress) → u256++` [.item-kind]#external#
+
+Returns the amount of a given `token` that can be released at the time of the call.
+
+[.contract-item]
+[[IVesting-vested_amount]]
+==== `[.contract-item-name]#++vested_amount++#++(token: ContractAddress, timestamp: u64) → u256++` [.item-kind]#external#
+
+Returns the total vested amount of a specified `token` at a given `timestamp`.
+
+[.contract-item]
+[[IVesting-release]]
+==== `[.contract-item-name]#++release++#++(token: ContractAddress) → u256++` [.item-kind]#external#
+
+Releases the amount of a given `token` that has already vested and returns that amount.
+
+Emits {AmountReleased} event.
+
+[#IVesting-Events]
+==== Events
+
+[.contract-item]
+[[IVesting-AmountReleased]]
+==== `[.contract-item-name]#++AmountReleased++#++(token: ContractAddress, amount: u256)++` [.item-kind]#event#
+
+Emitted when vested tokens are released to the beneficiary.
+
+[.contract]
+[[VestingComponent]]
+=== `++VestingComponent++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.17.0/packages/finance/src/vesting/vesting.cairo[{github-icon},role=heading-link]
+
+[.hljs-theme-dark]
+```cairo
+use openzeppelin_finance::vesting::VestingComponent;
+```
+
+[.contract-index]
+.Vesting Schedule Trait
```

Comment:
A note on top of this with a link to the `finance#vesting_schedule` guide can be helpful for users to understand the intention of this trait if they start on this page.

Reply:
Nice idea